packages/mgt/src/components/mgt-person/mgt-person.scss

```diff
@@ -5,376 +5,378 @@
+
+        &.presence-oof-dnd {
+          background-color: $presence-background-color;
+          border-color: $presence-color-dnd;
+        }
+
+        &.presence-busy {
+          background-color: $presence-color-dnd;
+          border-color: $presence-color-dnd;
+        }
+
+        &.presence-oof-busy {
+          background-color: $presence-background-color;
+          border-color: $presence-color-dnd;
+        }
+      }
+
+      .presence-offline::before {
+        content: $ms-icon-code-StatusCircleErrorX;
+        color: $presence-color-invisible;
+        font-weight: bold;
+        margin-left: calc(#{$avatar-size} * 0.07 - 2px);
+      }
+
+      .presence-available::before {
+        content: $ms-icon-code-SkypeCheck;
+        color: $presence-background-color;
+        margin-left: calc(#{$avatar-size} * 0.07 - 2px);
+      }
+
+      .presence-oof-available::before {
+        content: $ms-icon-code-SkypeCheck;
+        color: $presence-color-online;
+        margin-left: calc(#{$avatar-size} * 0.07 - 2px);
+      }
+
+      .presence-away::before {
+        content: $ms-icon-code-SkypeClock;
+        color: $presence-background-color;
+        margin-left: calc(#{$avatar-size} * 0.07 - 2px);
+      }
+
+      .presence-dnd::before {
+        content: $ms-icon-code-SkypeMinus;
+        color: $presence-background-color;
+        margin-left: calc(#{$avatar-size} * 0.07 - 2px);
+      }
+
+      .presence-oof-dnd::before {
+        content: $ms-icon-code-SkypeMinus;
+        color: $presence-color-dnd;
+        margin-left: calc(#{$avatar-size} * 0.07 - 2px);
+      }
+    }
+
+    &.initials {
+      color: $initials-color;
+      border-radius: $avatar-border-radius;
+      font-weight: 400;
+
+      & .initials-text,
+      & .contact-icon {
+        cursor: default;
+        margin-top: -1px;
+        font-size: calc(#{$avatar-size} * 0.4);
+      }
+
+      & .contact-icon {
+        display: inline;
+      }
+    }
+
+    &.small {
+      width: $avatar-size-s;
+      height: $avatar-size-s;
+
+      & .initials-text,
+      & .contact-icon {
+        font-size: calc(#{$avatar-size-s} * 0.4);
+      }
+      .user-presence {
+        left: calc(#{$avatar-size-s} - 8px);
+        top: calc(#{$avatar-size-s} - 8px);
+        width: 8px;
+        height: 8px;
+
+        .presence-basic {
+          width: 4px;
+          height: 4px;
+          border-radius: 4px;
+          padding: 0;
+
+          & .presence-oof-offline svg {
+            height: 0;
+            width: 0;
+          }
+        }
+
+        .presence-offline::before,
+        .presence-available::before,
+        .presence-oof-available::before,
+        .presence-away::before,
+        .presence-oof-away::before,
+        .presence-dnd::before,
+        .presence-oof-dnd::before,
+        .presence-oof-offline::before {
+          content: '';
+        }
+      }
+    }
+
+    img {
+      height: 100%;
+      width: 100%;
+      object-fit: cover;
```

Comment:
Have you tested in IE11? `object-fit` isn't supported there.

Reply:
Image is compressed in IE11. I tried some other CSS to replace object-fit to no success.